scootapi/locate.go

```diff
@@ -17,7 +17,8 @@ import (
 
 // Get the path of the file containing the address for scootapi to use
 func GetScootapiAddrPath() string {
-	return path.Join(os.Getenv("HOME"), ".cloudscootaddr")
+	optionalId := os.Getenv("SCOOT_ID") // Used to connect to a different set of scoot processes.
```

Comment:
ditto ?????? - if the explanation is put in the dev docs, then this code may be easier to understand.